chore: add upgrade file doc (#224)
sensiolabs/GotenbergBundle: UPGRADE-1.1.md

# UPGRADE FROM 1.0.0 to 1.1.0

## Configuration Changes

> * Added new `sensiolabs_gotenberg.version` to add warning if trying to use a non-yet available feature from Gotenberg.

Reviewer: ```suggestion
* Added new `sensiolabs_gotenberg.version` to add warning when trying to use a feature not yet available in Gotenberg.
```

> * Added new `sensiolabs_gotenberg.version` to add warning when trying to use a feature not yet available in Gotenberg.

Reviewer 2: We could also specify that the version will be retrieved with a HTTP call if not set.

Author: as it was kind the reason of my first issue I think its a good idea 👍🏻

Reviewer 2: ```suggestion
* Added new `sensiolabs_gotenberg.version` to add warning when trying to use a feature not yet available in Gotenberg.
  > [!NOTE]
  > If no version is explicitly defined in the configuration, the bundle will perform an additional HTTP call to the 
  > Gotenberg API to resolve the compatible version automatically.
```
?
